HW_Sem_3/Task1.py

```diff
@@ -0,0 +1,15 @@
+# Задайте список из нескольких чисел. 
+# Напишите программу, которая найдёт сумму элементов списка,
+#  стоящих на нечётной позиции.
+# Пример:
+# - [2, 3, 5, 9, 3] -> на нечётных позициях элементы 3 и 9, ответ: 12
+
+from random import randint
+
+num = int(input("Введите количество элементов: "))
+num_list = [randint(0, num) for _ in range(num)]
+sum_list = [num_list[i] for i in range(1, num, 2)]
+num_sum = sum(sum_list)
```

Comment:
```suggestion
num_sum = sum(sum_list[1::2])
```
Можно и так. И на одну строку короче.